Hunting Queries/LAQueryLogs/NewServicePrincipal.yaml

```diff
@@ -0,0 +1,30 @@
+id: 98e4df23-7bd2-480d-814a-a03f77efc670
+name: New ServicePrincipal running queries
+description: |
+  'This hunting query looks for new Service Principals running queries that have not previously been seen running queries.'
+requiredDataConnectors:
+  - connectorId: CustomConnector
```

Comment:
Lets use connectorId: "AzureMonitor(Query Audit)" for now, it may change slightly but should be under Azure Monitor.